src/model/article-vor.v4.yaml

```diff
@@ -2,18 +2,9 @@ $schema: http://json-schema.org/draft-04/schema#
 title: Article VoR
 type: object
 allOf:
-  - $ref: ../common/article-vor.v1.yaml
```

Comment:
I was having trouble on Tuesday trying to find the right spec to validate a snippet against.

It looked like I could use the full article spec to validate a snippet but I was getting validation errors on otherwise valid data. Is the full article spec what I should be using? Because I'll persevere through the validation errors and figure out what they're actually trying to tell me.

Reply:
When I validate a snippet I always do it in the context of a list, so I would wrap the snippet in an array and use https://github.com/elifesciences/api-raml/blob/develop/dist/model/article-list.v1.json

Reply:
That's clever, I thought about doing something similar but figured I was just looking in the wrong place.

My tests are working with this now but I wonder what it would take to tease the article snippet out into it's own schema - or if it's even worth it.

Reply:
> My tests are working with this now but I wonder what it would take to tease the article snippet out into it's own schema - or if it's even worth it.

You could do it, it would require another script similar to `./compile.js` targeting the `./src/snippets` folder. Personally, I don't think it's worth it. `article-list.v1.json` is sufficient.